pkg/registry/apis/identity/register.go

```diff
@@ -13,63 +13,70 @@ import (
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	common "k8s.io/kube-openapi/pkg/common"
 
-	identity "github.com/grafana/grafana/pkg/apimachinery/apis/identity/v0alpha1"
-	identityapi "github.com/grafana/grafana/pkg/apimachinery/identity"
+	"github.com/grafana/grafana/pkg/apimachinery/identity"
+	identityv0 "github.com/grafana/grafana/pkg/apis/identity/v0alpha1"
 	grafanarest "github.com/grafana/grafana/pkg/apiserver/rest"
 	"github.com/grafana/grafana/pkg/infra/db"
 	"github.com/grafana/grafana/pkg/registry/apis/identity/legacy"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/serviceaccount"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/sso"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/team"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/user"
 	"github.com/grafana/grafana/pkg/services/apiserver/builder"
 	"github.com/grafana/grafana/pkg/services/featuremgmt"
+	"github.com/grafana/grafana/pkg/services/ssosettings"
 	"github.com/grafana/grafana/pkg/storage/legacysql"
 )
 
 var _ builder.APIGroupBuilder = (*IdentityAPIBuilder)(nil)
 
 // This is used just so wire has something unique to return
 type IdentityAPIBuilder struct {
-	Store legacy.LegacyIdentityStore
+	store      legacy.LegacyIdentityStore
+	ssoEnabled bool
+	ssoService ssosettings.Service
```

Comment:
```suggestion
	Store      legacy.LegacyIdentityStore
	SSOService ssosettings.Service
```

we can do ssoEnabled as `SSOService != nil` -- keeping this public will keep things working here https://github.com/grafana/grafana-enterprise/pull/7068/files#diff-7027320bf121b6a4f613cb1268191f4c0ba0249eaf4fbb19d4acc041d8d19ae2R467